<launch>
    <param name="device" type="str" value="cpu" />
    <param name="prefix" type="str" value="$(find mxnet)/models/gluon_resnet18/final" />
    <param name="epoch" type="int" value="0" />
    <param name="image_size" type="int" value="224" />
    <param name="classes" type="str" value="$(find mxnet)/models/mxnet_classes.txt" />
    
    <param name="input_topic" type="str" value="/usb_cam/image_raw" />
    <param name="detection_topic" type="str" value="/image_detection" />

    <node name="object_detector" pkg="mxnet" type="detect.py" output="screen"/>
</launch>